examples/Radiation.ipynb

```diff
@@ -111,9 +111,67 @@
     "\n",
     "m.add_control(search)\n",
     "\n",
-    "\n",
-    "m"
+    "\n"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "049eafa5-e939-442a-a24e-cfd69ef3372d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipyflex import FlexLayout\n",
```

Comment:
Do you mind adding `ipyflex` to the binder requirements here: https://github.com/jupyter-widgets/ipyleaflet/blob/master/environment.yml ? That way this example will still work on Binder :)